proposals/artifacts.md

```diff
@@ -0,0 +1,58 @@
+# OCI artifact project proposal
+
+## Abstract
+Container registries, implementing the [distribution-spec][distribution-spec], provide reliable, highly scalable, secured storage services for container images. Customers either use a cloud provider implementation, vendor implementations, or instance the open source implementation of distribution. They configure security and networking to assure the images in the registry are locked down and accessible by the resources required. Cloud providers and vendors often provide additional values atop their registry implementations from security to productivity features. 
+
+Applications and services typically require additional artifacts to deploy and manage, including [helm](https://helm.sh) for deployment and [Open Policy Agent (OPA)](https://github.com/open-policy-agent/opa/issues/1413) for policy enforcement.
+
+Utilizing the [manifest][image-manifest] and [index][image-index] definitions, new artifacts can be stored and served using the [distribution-spec][distribution-spec] without changing the actual distribution spec. This repository will provide  a reference for artifact authors and registry implementors for supporting new artifact types with the existing implementations of distribution.
```

Comment:
/s/the [/the OCI [/

Reply:
/s/new artifacts/new artifact types/

Reply:
/s/supporting new/supporting these new/

Reply:
Hey Mike, I'm not sure I understood the convention make the changes you're requesting. Can you confirm the PR change?

Reply:
I believe I captured everything. Please un-resolve if I missed something.